src/main/resources/org/jenkinsci/plugins/pollscm/PollNowAction/action.jelly

```diff
@@ -34,7 +34,12 @@ THE SOFTWARE.
                 href="${h.getActionUrl(it.url,action)}/polling" onclick="${'return poll_' + id + '(this)'}" post="true"/>
         <script>
             function poll_${id}(a) {
-                new Ajax.Request(a.href);
+                fetch(a.href, {
+                  method: "post",
+                  headers: crumb.wrap({
+                    "Content-Type": "application/x-www-form-urlencoded",

```

Comment:
You don't need to set a content type here since the body of the request is empty.

Reply:
Thanks!  Included in 5648e66bad3f925a296533f9017b97aa01e624d9